packages/theme-language-server-common/src/completions/providers/LiquidTagsCompletionProvider.spec.ts

```diff
@@ -44,6 +44,15 @@ describe('Module: LiquidTagsCompletionProvider', async () => {
     await expect(provider).to.complete('{% form "cart", cart %} ... {% end', ['endform']);
   });
 
+  it('should not complete literal `liquid` tag', async () => {
+    await expect(provider).to.complete(
+      `{% liquid
+        echo 'hello'
+        e`,
+      ['echo'],
```

Comment:
Would it make sense to do something like `expect(provider).not.to.complete('code', ['endliquid'])`? Or maybe that would end up with a false positive. Almost wish there was a slightly different assertion that was like:

```ts
const completionOptions = provider.complete('code');
expect(completionOptions).not.toContain('endliquid')`
```

Reply:
Funny you say this, in my Liquid Tag snippet completion PR, I modified the assertion to work with top level async matchers. I think that would make it possible with the not to complete with an expect.arrayContaining().

https://github.com/Shopify/theme-tools/pull/507/files#diff-77716827f5743abd97b0ab800c5b78475604beb65b8b5f21a56a5f18d8b82765

But, I mean... this test looks fine to me. The list of tags here is hardcoded at the top of the file, so this test would only break by manual intervention.

Reply:
> The list of tags here is hardcoded at the top of the file, so this test would only break by manual intervention.
nice! didn't realize that